Sources/Tracing/Span.swift

```diff
@@ -269,7 +269,7 @@ public enum SpanAttribute: Equatable {
         .int(0)
     }
 
-    internal var anyValue: Any {
+    public var anyValue: Any {
```

Comment:
One way to get around this would be to make this property public and then the conversions would have to be...

```
            if let route = httpSpan.attributes["http.route"] {
                metadata["http_route"] = "\(route.anyValue)"
            }
```

This is pretty meh though; so continue reading